src/util.ts

```diff
@@ -88,3 +89,19 @@ export function createOutputRetainingCompilerHost(
     outputFiles.set(fileName, content);
   }
 }
+
+/**
+ * True if we are running on Windows (32 or 64 bit).
+ */
+export const isWindows: boolean = os.platform() === 'win32';
+
+/**
+ * Returns the input with line endings normalized to '\n', when running on Windows.
+ * Returns the string as is, for any other OS.
+ */
+export function normalizeLineEndings(input: string): string {
+  if (isWindows)
```

Comment:
Just drop it, it won't save substantial time. It is adding some more complexity/branchiness to test code, which we generally want to avoid (bugs in test setup code are the worst...).